src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
> but what's wrong in OCamlPro offering this service and some sysadmin time ?

OCamlPro's website has been 404ing more than once with stuff hosted there (the last occurence being that [one](https://github.com/ocaml/opam/issues/2438)). We don't want to wait for an OCamlPro employee to show up to be able to fix these problems. I'm sure your sysadmin time will be gladly appreciated on opam.ocaml.org.